build_tools/autogen_ltc_backend.py

```diff
@@ -38,7 +38,7 @@ def reindent(text, prefix=""):
 @dataclass(frozen=True)
 class GenMlirLazyIr(torchgen.dest.GenLazyIR):
     def isOptionalCType(self, arg):
-        return str(type(arg)) == "<class 'torchgen.api.types.OptionalCType'>"
+        return str(type(arg)) == "<class 'torchgen.api.types.types.OptionalCType'>"
```

Comment:
Is there a test to catch changes like this? It would be the best if we can have CI to detect any breaking changes.

Reply:
@ke1337 we have such tests and they catched the problem. See https://github.com/llvm/torch-mlir/issues/1708 for more details.